Dialog/PatchManagerTableView.cpp

```diff
@@ -0,0 +1,164 @@
+#include "PatchManagerTableView.h"
+#include <cassert>
+#include <QModelIndex>
+
+/// <summary>
+/// Construct an instance of the PatchManagerTableView.
+/// </summary>
+/// <param name="parent">
+/// The parent QWidget.
+/// </param>
+PatchManagerTableView::PatchManagerTableView(QWidget *param) : QTableView(param),
+    EntryTableModel(this)
+{
+    // Configure the table
+    setModel(&EntryTableModel);
+    setSelectionBehavior(SelectionBehavior::SelectRows);
+
+    // Populate the table
+    QVector<struct PatchEntryItem> patches = PatchUtils::GetPatchesFromROM();
+    foreach(struct PatchEntryItem patch, patches)
+    {
+        EntryTableModel.AddEntry(patch);
+    }
+
+    // TEST
```

Comment:
delete if don't need forever

Reply:
I will delete this (and put a TODO here) in the PR where this feature is completed